beacon-chain/core/blocks/block_operations.go

```diff
@@ -864,7 +867,9 @@ func ProcessDeposit(
 			domain := helpers.Domain(beaconState, helpers.CurrentEpoch(beaconState), params.BeaconConfig().DomainDeposit)
 			depositSig := deposit.Data.Signature
 			if err := verifySigningRoot(deposit.Data, pubKey, depositSig, domain); err != nil {
-				return nil, fmt.Errorf("could not verify deposit data signature: %v", err)
+				//ignore this error as in the spec pseudo code
+				log.Errorf("could not verify deposit data signature: %v", err)
```

Comment:
```suggestion
				log.Errorf("Skipping deposit: could not verify deposit data signature: %v", err)
```

It would be really nice to have a prometheus metric for this event. Could you follow up with one in another PR?

Reply:
I never worked with Prometheus metrics before so I would have to look into that, but I can definitely try.